app/services/audit/AuditService.scala

```diff
@@ -36,19 +37,19 @@ class AuditService @Inject()(connector: AuditConnector, @Named("appName") appNam
 
   def auditShutMucr(providerId: String, mucr: String, result: String)(implicit hc: HeaderCarrier): Future[AuditResult] =
     audit(
-      AuditTypes.AuditShutMucr,
+      AuditType.AuditShutMucr,
       Map(EventData.providerId.toString -> providerId, EventData.mucr.toString -> mucr, EventData.submissionResult.toString -> result)
     )
 
-  def auditDisassociate(providerId: String, ucr: String, result: String)(implicit hc: HeaderCarrier): Future[AuditResult] =
+  def auditDisassociate(providerId: String, ducr: String, result: String)(implicit hc: HeaderCarrier): Future[AuditResult] =
```

Comment:
are we here able to dissociate only `DUCR`s ?

Reply:
Pretty sure this should be `ucr` as it was

Reply:
Must have changed that during rebasing. Will change it back.